content/en/blog/_posts/2021-04-08-kubernetes-release-1.21.md

```diff
@@ -0,0 +1,124 @@
+
+Currently, when a node shuts down, pods do not follow the expected termination lifecycle and are not shutdown gracefully. This can introduce problems with a lot of different workloads. Going forward, Kubelet will be able to propagate the shutdown signal to pods ensuring they can terminate as gracefully as possible.
+
+### PersistentVolume Health Monitor
+Physical Volumes (PV) are commonly used in applications to get local, file-based storage. They can be used in many different ways and help users migrate applications without needing to re-write storage backends.
+
+Kubernetes 1.21 has a new alpha feature which allows PVs to be monitored for health of the volume and marked accordingly if the volume becomes unhealthy. Workloads will be able to react to the health state to protect data from being written or read from a volume that is unhealthy.
+
+### Reducing Kubernetes Build Maintenance
+Previously Kubernetes has maintained multiple build systems. This has often been a source of friction and complexity for new and current contributors.
+
+Over the last release cycle, a lot of work has been put in to simplify the build process, and standardize on the native Golang build tools. This should empower broader commmunity maintenance, and lower the barier to entry for new contributors.
+
+## Major Changes
+
+### PodSecurityPolicy Deprecation
+In Kubernetes 1.21, PodSecurityPolicy is deprecated. As with all Kubernetes feature deprecations, PodSecurityPolicy will continue to be available and fully-functional for several more releases. PodSecurityPolicy, previously in the beta stage, is planned for removal in Kubernetes 1.25.
+
+What's next? We're developing a new feature, tentatively called PodIsolationPolicy, that covers the key PodSecurityPolicy use cases with greatly improved ergonomics and maintainability. To learn more, check out the PSP feature blog [here]().
+
+### TopologyKeys Deprecation
+The service field topologyKeys is being deprecated and replaced with a new feature called topology-aware hints, which will be released as alpha in Kubernetes 1.21. You can read more details about the proposed changes in the [KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/2433-topology-aware-hints/README.md)
+
+## Other Updates
+
+### Graduated to Stable
+
+* [EndpointSlice](https://github.com/kubernetes/enhancements/issues/752)
+* [Add sysctl support](https://github.com/kubernetes/enhancements/issues/34)
+* [PodDisruptionBudgets](https://github.com/kubernetes/enhancements/issues/85)
+
+### Notable Feature Updates
+
+* [External client-go credential providers](https://github.com/kubernetes/enhancements/issues/541) - alpha in 1.21
```

Comment:
I think these are [beta](https://github.com/kubernetes/enhancements/issues/541#issuecomment-773827118). Is that right?

Reply:
Yep, my bad.